package.json

```diff
@@ -27,6 +27,7 @@
 	},
 	"dependencies": {
 		"commander": "4.0.1",
+		"comment-json": "^4.1.0",
```

Comment:
Please use exact dependency. I wonder why https://github.com/Bessonov/set-project-references/blob/master/.npmrc#L1 had not any effect.

Reply:
that's a really good question. i had to delete the package-lock that was created as well.. so weird.. no worries i'll edit